customer-satisfaction/steps/train_model.py

```diff
@@ -1,53 +1,60 @@
 import logging
+from typing import Annotated
 
 import mlflow
 import pandas as pd
-from model.model_dev import ModelTraining
+
+from model.model_dev import ModelTrainer
 from sklearn.base import RegressorMixin
 from zenml.client import Client
-from zenml.steps import Output, step
+from zenml import step
+from zenml import ArtifactConfig
 
-from .config import ModelNameConfig
 
 experiment_tracker = Client().active_stack.experiment_tracker
 
 
-@step(experiment_tracker=experiment_tracker.name)
+@step(enable_cache=False, experiment_tracker=experiment_tracker.name)
 def train_model(
     x_train: pd.DataFrame,
     x_test: pd.DataFrame,
     y_train: pd.Series,
     y_test: pd.Series,
-    config: ModelNameConfig,
-) -> Output(model=RegressorMixin):
+    model_type: str = "lightgbm",
+    do_fine_tuning: bool = True
+) -> Annotated[
+    RegressorMixin, ArtifactConfig(name="model", is_model_artifact=True)

```

Comment:
What I like to do is call this `sklearn_regressor` or something so that it isnt confused by the zenml model / model version concept.

Reply:
that makes a lot of sense